README.md

```diff
@@ -124,3 +133,105 @@ Path parameters
 | Parameter | Type | Function |
 |:----------|:-----|:---------|
 | ECNumber | String | The EC number you want ligands associated with |
+
+
+#### Docking
+
+###### To perform docking, start by calling the ```POST /dock/dockligand``` endpoint.
+###### Once you receive a 200 status and a job ID, continue to query the ```GET /dock/dockligand``` endpoint
+###### Once you receive a 200 status, active site data, binding affinities and job ID, call ```GET /dock/retrievefile```
+###### Continue to call ```GET /dock/retrievefile``` until a 200 status is returned.
+###### If you would like to receive the completed docked file again, you may call the ```GET /dock/retrievefile``` endpoint
+###### The location of the AutoDock Vina and OpenBabel deployments may be configured in the ```application.properties``` file
+
+#### Docking API
+
+<a name="dock-ligand"></a>
+##### POST /dock/dockligand
+###### Convert the supplied ligand and protein to pdbqt format using OpenBabel then start an AutoDock Vina job
+
+##### Form Parameters
+
+| Parameter | Type | Function |
+|:----------|:-----|:---------|
+| macromolecule | File | Protein file for docking |
+| ligand | File | Ligand file for docking |
+| ligandID | String | Optional ligand ID. If ligand file is not provided, ligand ID will be used to download ligand from the PDB |
+| macromoleculeID | String | Optional macromolecule ID. If macromolecule file is not provided, macromolecule ID will be used to download macromolecule from the PDB |
+| center_x | number | Center of docking area in the x axis |
+| center_y | number | Center of docking area in the y axis |
+| center_z | number | Center of docking area in the z axis |
+| size_x | number | Size of docking area in the x axis |
+| size_y | number | Size of docking area in the y axis |
+| size_z | number | Size of docking area in the z axis |
+
+##### Returns
+
+###### 200 OK - When job has been completed successfully
+
+| Parameter | Type | Function |
```

Comment:
Its a bit difficult to read this when the input and return tables are identical. Could you change them a bit - maybe just change the name of the "parameters" column to something else

Reply:
I have changed 'Parameter' to 'Field' on all return tables to differentiate them.